app/assets/stylesheets/elements/_dropdown.scss

```
@@ -38,6 +38,10 @@
   overflow-y: auto;
   z-index: 100;
 
+  @media (max-width: 999px) {
+    right: -70px;
+  }
```

Let's keep this mobile first and use the media-min mixin to keep things a little more consistent.

```suggestion
  overflow-y: auto;
  z-index: 100;
  right: -70px;

  @include media-min(mbl) {
    right: 0px;
  }
```

---

`@media (max-width: 999px)` is already used in another place in this file, adding a `@include media-min(mbl)` seems less consistent to me. i take it the mixin is used in the other .scss files dealing with PC/mobile differences?